### DISCUSSION TITLE
【Unity】后续更换.netcore导致gc流程修改会影响xil2cpp吗

### DISCUSSION THREAD

**Discussion by danggui1995** [Tencent/puerts#1858] [General]
感觉不是很远了

**Comment by chexiongsheng** [comment]
如果gc改为会改移动对象，或者说对象指针会变的话，目前xil2cpp用不了，但到时我也会做相应调整。
我现在有对gc做检查，如果不是用BOEHM会编译不过：https://github.com/Tencent/puerts/blob/master/unity/Assets/core/upm/Editor/Resources/puerts/xil2cpp/Puerts_il2cpp.cpp.txt#L38

**Comment by chexiongsheng** [comment]
你们项目有在用xil2cpp吗？
我重构xil2cpp时看代码觉得目前的xil2cpp实现会有这样的问题，你们有没碰到？https://github.com/Tencent/puerts/issues/1861

**Reply by danggui1995** [reply]
没开始用  我是自己在折腾
